search_single_spectrum/tools/search_single_spectrum/foodomics_track.py

```diff
@@ -46,38 +47,73 @@ def calculate_enrichment(file_occurrent_df, metadata_df):
 
     return pd.DataFrame(output_list)
 
-def metadata_file_matches(file_occurrent_df):
+
+def metadata_file_matches(matches_df):
+    """
+    Create filtered metadata table for MASST matched files
+    :param matches_df: filtered MASST matches in GFOP files : pandas dataframe
+    :return: filtered metadata
+    """
     # Filter to appropriate dataset
-    file_occurrent_df = file_occurrent_df[file_occurrent_df["dataset_id"] == "MSV000084900"]
+    matches_df = matches_df[matches_df["dataset_id"] == "MSV000084900"]
```

Comment:
Isn't the table already filtered to this dataset id? Line 88

Reply:
yes. Removed it